File: programmers/js/basic/181865.js
// 간단한 식 계산하기
function solution(binomial) {
  const [a, op, b] = binomial.split(" ");

  switch (op) {
    case "+":
      return Number(a) + Number(b);
    case "-":
      return Number(a) - Number(b);
    case "*":
      return Number(a) * Number(b);
  }
}

File: programmers/java/basic/181865.java
// 간단한 식 계산하기

class Solution {
    public int solution(String binomial) {
        String[] splited = binomial.split(" ");
        int a = Integer.parseInt(splited[0]);
        int b = Integer.parseInt(splited[2]);
        String operator = splited[1];
        
        if (operator.equals("+")) {
            return a + b;
        } else if (operator.equals("-")) {
            return a - b;
        } else {
            return a * b;
        }
    }
}